optimum/exporters/tasks.py

```diff
@@ -437,6 +437,16 @@ class TasksManager:
             "image-classification",
             onnx="MobileViTOnnxConfig",
         ),
+        "mobilenet-v1": supported_tasks_mapping(
```

Comment:
```suggestion
        "mobilenet_v1": supported_tasks_mapping(
```

to match the config `model_type`

Reply:
as you told me this might break existing models such as data2vec-text etc, so let's keep it as it is